data/gegno/ships.txt

```diff
@@ -0,0 +1,954 @@
+	thumbnail thumbnail/coesite
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Gegno Civilian"
+			"Scin Adjutant"
+		cost 3120000
+		mass 245
+		bunks 26
+		"cargo space" 60
+		drag 4
+		"engine capacity" 120
+		"fuel capacity" 400
+		"heat dissipation" 0.7
+		hull 6450
+		"outfit space" 264
+		"required crew" 9
+		shields 0
+		"weapon capacity" 110
+		weapon
+			"blast radius" 40
+			"shield damage" 240
+			"hull damage" 460
+			"hit force" 440
+	outfits
+		"Astuit Battery" 2
+		"C7 Brawl Cell"
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"R02 Battlezone Battery" 2
+		"SC-12 Plasma Thruster"
+		"SC-14 Plasma Steering"
+	engine 46.5 92.5
+	engine -46.5 92.5
+	turret -25 -16.5 "Astuit Battery"
+	turret 25 -16.5 "Astuit Battery"
+	leak leak 60 50
+	leak flame 50 80
+	explode "medium explosion" 6
+	explode "small explosion" 24
+	explode "tiny explosion" 18
+	"final explode" "final explosion small"
+	description "Coesites are smaller warships that the Scin use as screening vessels during battle with the Vi. They are speedy and carry 3 turret mounts which allows them to always be orbiting and firing on the move. Coesites are the designated platform for the Scin's anti-shell turrets, allowing them to take out shells ahead of their larger ships to reduce incoming fire and turn the tide of war."
+
+ship Tridymite
+	sprite ship/tridymite
+	thumbnail thumbnail/tridymite
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Gegno Civilian"
+			"Scin Architect"
+			"Scin Hoplologist"
+		cost 1960000
+		mass 225
+		bunks 20
+		"cargo space" 40
+		drag 4
+		"engine capacity" 86
+		"fuel capacity" 400
+		"heat dissipation" 0.7
+		hull 5890
+		"outfit space" 190
+		"required crew" 9
+		shields 0
+		"weapon capacity" 62
+		weapon
+			"blast radius" 35
+			"shield damage" 220
+			"hull damage" 430
+			"hit force" 410
+	outfits
+		"C7 Brawl Cell"
+		"Guile Anti-Missile"
+		Hyperdrive
+		"R02 Battlezone Battery"
+		"SC-12 Plasma Thruster"
+		"SC-14 Plasma Steering"
+	engine 38.5 73
+		zoom 0.9
+	engine -38.5 73
+		zoom 0.9
+	engine 0 67.5
+	turret 0 18.5 "Guile Anti-Missile"
+	leak leak 50 40
+	leak flame 40 70
+	explode "medium explosion" 3
+	explode "small explosion" 19
+	explode "tiny explosion" 14
+	"final explode" "final explosion small"
+	description "Using a similar yet smaller frame to the Coesite, the Tridymite was designed as the production platform for the Gegno Scin's new Guile system. As its younger sibling, the Tridymite is a smaller escort vessel that provides anti-cannon screening for Scin fleets, while Coesites are armed with more offensive weaponry."
+
+ship Felsic
+	sprite ship/felsic
+	thumbnail thumbnail/felsic
+	swizzle 0
+	attributes
+		category "Light Freighter"
+		licenses
+			"Gegno Civilian"
+			"Scin Architect"
+			"Scin Hoplologist"
+		cost 2342000
+		mass 565
+		bunks 78
+		"cargo space" 160
+		drag 5.9
+		"engine capacity" 106
+		"fuel capacity" 500
+		"heat dissipation" 0.7
+		hull 11000
```

Comment:
They don't lose a lot of hull compared to the Schists for being way better in every other way. Are the Scin ships supposed to be far superior to the Vi?

Reply:
Scin ships are supposed to be technologically superior, but I think like you said maybe it should loose more hull. Though I'd be worried they are worse off in combat due to that, so maybe not a lot of hull.

Strictly speaking, the Felsic is the best re-design of the Dolomite over the Schist, which is only focused on a cargo role.